src/client.ts

```diff
@@ -125,11 +150,11 @@ export const XmtpClientInterface: Client = {
 };
 
 const processMessage = async (
-  message: DecodedMessage,
+  message: DecodedMessage<any>,
   conversation: Conversation
 ) => {
   try {
-    const text = message?.content?.text ?? "";
```

Comment:
is this right?

Reply:
yes, here is an example of DecodedMessage response with some redacted info

```
Processing message DecodedMessage {
  content: 'gm',
  contentType: ContentTypeId {
    authorityId: 'xmtp.org',
    typeId: 'text',
    versionMajor: 1,
    versionMinor: 0
  },
  conversationId: '...',
  deliveryStatus: 'published',
  fallback: undefined,
  compression: undefined,
  id: '...,
  kind: 'application',
  parameters: { encoding: 'UTF-8' },
  senderInboxId: '...',
  sentAt: 2025-04-22T16:50:45.447Z,
  sentAtNs: 1745340645447197000
}
```